kong/plugins/file-log/schema.lua

```diff
@@ -10,8 +28,10 @@ return {
         fields = {
           { path = { type = "string",
                      required = true,
-                     match = [[^[^*&%%\`]+$]],
-                     err = "not a valid filename",
+                     match = path_pattern,
```

Comment:
I think what will happen is customers will be unable to update/patch existing file log plugins unless they make sure the existing config.path matches the new prefix.

We may want to have a migration/cli script that finds and prints out the existing plugins that need to be updated to follow the new rules. 

Let's discuss ways we can help customers who find themselves in this situation.

Reply:
Yes, this change seems to be a bit too much user impacting. As a matter of fact, more impacting to the sandboxing change we did a while ago. Need so find a better way to not surprise the customers after the fact.